odl/discr/discr_ops.py

```diff
@@ -54,74 +61,81 @@ def finite_diff(f, out=None, axis=0, dx=1.0, edge_order=2,
     f : `array-like`
          An N-dimensional array
     out : `numpy.ndarray`, optional
-         An N-dimensional array to which the output is written.
+         An N-dimensional array to which the output is written
     axis : `int`, optional
-        The axis along which the partial derivative is evaluated. Default: 0
+        The axis along which the partial derivative is evaluated
     dx : `float`, optional
-        Scalar specifying the sample distances in each dimension ``axis``.
-        Default distance: 1.0
-    edge_order : {1, 2}, optional
-        First-order accurate differences (1) can be used at the boundaries
-        if no zero padding is used. Default edge order: 2
-    zero_padding : `bool`, optional
-        Implicit zero padding. Assumes values outside the domain of ``f`` to be
-        zero. Default: `False`
+        Scalar specifying the distance between sampling points along ``axis``
     method : {'central', 'forward', 'backward'}, optional
-        The method that should be used.
+        Finite difference method which is used in the interior of the domain
+         of ``f``.
+    padding : `float` or 'edge', optional
```

Comment:
If `None` one-sided forward or backward differences are used at the boundary. It's described in doc string, but it was in the wrong place in the Parameters section